### ISSUE TITLE
Absturzbericht von Dynamo 2.5.0.7460

### ISSUE THREAD

**Issue by angelromosandoval** [DynamoDS/Dynamo#11875]
# Issue Description
Please fill in the following information to help us reproduce the issue:

### What did you do?
I tried to add a new folder to the packages folders

### What did you expect to see?
That Dynamo would add the chosen folder

### What did you see instead?
Suddenly this error

### What packages or external references (if any) were used?
none

### Stack Trace
```
(From the Dynamo crash window select 'Details' -> 'Copy' and paste here)
```

---
OS: `Microsoft Windows NT 10.0.19041.0`
CLR: `4.0.30319.42000`
Dynamo: `2.5.0.7460`

**Comment by QilongTang** [comment]
@angelromosandoval Do you remember what kind of error it was? I guess it was a hard crash you were running into?

**Comment by angelromosandoval** [comment]
Hi,

Each time I try to remove a folder of the packages folders, receive this
error.
I can manually remove changing the Dynamosettings.xml but I would like to
avoid this error.

Thanks

**Comment by QilongTang** [comment]
Thanks @angelromosandoval Do you mind sending us the stack trace of the crash so our team can digest how to avoid such error for you in the future release?

**Comment by angelromosandoval** [comment]
Die Datei oder Assembly
"file:///C:\Users\asandoval\AppData\Roaming\Dynamo\Dynamo
Revit\2.5\packages\CivilConnection2021\bin\Autodesk2021.dll" oder eine
Abhängigkeit davon wurde nicht gefunden. Der Vorgang wird nicht
unterstützt. (Ausnahme von HRESULT: 0x80131515)

   bei System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName,
String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint,
StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean
throwOnFileNotFound, Boolean forIntrospection, Boolean
suppressSecurityChecks)
   bei
System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName
assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly,
StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean
throwOnFileNotFound, Boolean forIntrospection, Boolean
suppressSecurityChecks)
   bei System.Reflection.RuntimeAssembly.InternalLoadFrom(String
assemblyFile, Evidence securityEvidence, Byte[] hashValue,
AssemblyHashAlgorithm hashAlgorithm, Boolean forIntrospection, Boolean
suppressSecurityChecks, StackCrawlMark& stackMark)
   bei System.Reflection.Assembly.LoadFrom(String assemblyFile)
   bei Dynamo.PackageManager.PackageLoader.TryLoadFrom(String filename,
Assembly& assem)
   bei Dynamo.PackageManager.Package.EnumerateAssembliesInBinDirectory()
   bei
Dynamo.PackageManager.PackageLoader.<>c.<LoadPackages>b__52_0(Package x)
   bei System.Linq.Enumerable.<SelectManyIterator>d__17`2.MoveNext()
   bei System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   bei Dynamo.Models.DynamoModel.LibraryLoaded(Object sender,
LibraryLoadedEventArgs e)
   bei System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   bei
Dynamo.Engine.LibraryServices.OnLibrariesImported(LibraryLoadedEventArgs e)
   bei Dynamo.PackageManager.PackageLoader.LoadPackages(IEnumerable`1
packages)
   bei Dynamo.PackageManager.PackageLoader.LoadAll(LoadPackageParams
loadPackageParams)
   bei
Dynamo.PackageManager.PackageLoader.LoadCustomNodesAndPackages(LoadPackageParams
loadPackageParams, CustomNodeManager customNodeManager)
   bei Dynamo.ViewModels.PackagePathViewModel.CommitChanges(Object param)
   bei
Dynamo.Wpf.Views.PackageManager.PackagePathView.OnOkButtonClicked(Object
sender, RoutedEventArgs e)
   bei System.Windows.EventRoute.InvokeHandlersImpl(Object source,
RoutedEventArgs args, Boolean reRaised)
   bei System.Windows.UIElement.RaiseEventImpl(DependencyObject sender,
RoutedEventArgs args)
   bei System.Windows.Controls.Primitives.ButtonBase.OnClick()
   bei System.Windows.Controls.Button.OnClick()
   bei
System.Windows.Controls.Primitives.ButtonBase.OnMouseLeftButtonUp(MouseButtonEventArgs
e)
   bei System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler,
Object target)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target,
RoutedEventArgs routedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(Object source,
RoutedEventArgs args, Boolean reRaised)
   bei System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender,
RoutedEventArgs args, RoutedEvent newEvent)
   bei System.Windows.UIElement.OnMouseUpThunk(Object sender,
MouseButtonEventArgs e)
   bei System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler,
Object target)
   bei System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target,
RoutedEventArgs routedEventArgs)
   bei System.Windows.EventRoute.InvokeHandlersImpl(Object source,
RoutedEventArgs args, Boolean reRaised)
   bei System.Windows.UIElement.RaiseEventImpl(DependencyObject sender,
RoutedEventArgs args)
   bei System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   bei System.Windows.Input.InputManager.ProcessStagingArea()
   bei System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   bei System.Windows.Input.InputProviderSite.ReportInput(InputReport
inputReport)
   bei System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr
hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x,
Int32 y, Int32 wheel)
   bei System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr
hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd,
Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam,
IntPtr lParam, Boolean& handled)
   bei MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate
callback, Object args, Int32 numArgs)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object
source, Delegate callback, Object args, Int32 numArgs, Delegate
catchHandler)

**Comment by QilongTang** [comment]
FYI: @jasonstratton This looks like another package loading crash we can follow up and fix in Dynamo 2.13.0

**Comment by mjkkirschner** [comment]
definitely this should not crash - it should log a notification - this is issue is easily searchable by the hresult code - a binary is blocked or on a network drive.
this issue has been reported many times - it would be nice if when we catch this hresult code we let the user know what to do in Dynamo instead of making them post here.

https://stackoverflow.com/questions/34400546/could-not-load-file-or-assembly-operation-is-not-supported-exception-from-hres
